style.css

```diff
@@ -0,0 +1,212 @@
+/*===========================================
+      TABLE OF CONTENTS
+=======================================
+
+1. NAVBAR
+2. FORM BODY
+3. FOOTER
+
+*/
+
+@import url("https://fonts.googleapis.com/css?family=Montserrat&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Nunito&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Nunito", sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+/*============================================
+      NAVBAR
+=============================================*/
+nav {
+  background: #eceef1;
+  display: flex;
+  justify-content: flex-end;
+  padding: 10px 30px;
+}
+
+nav button {
+  background: #0077c5;
+  border-radius: 4px;
+  border: 1px solid #4c4c4c;
+  width: 102.36px;
+  height: 34px;
+  color: #ffffff;
+  font-size: 13px;
+}
+
+nav p {
+  font-size: 12px;
+  color: #505050;
+  padding: 3px 0px;
+  font-family: Nunito;
+}
+
+.nav-desc {
+  padding-right: 6px;
+}
+
+/*===============================================
+      FORM BODY
+=================================================*/
+
+.icons-set span {
+  font-family: Montserrat, sans-serif;
+  color: #8d9096;
+}
+
+h2 {
+  color: #4c4c4c;
+  line-height: 1.1;
+  font-size: 24px;
+  padding: 30px 0px;
+}
+
+section {
+  background-color: #fff;
+  border: 1px solid #c7c7c7;
+  font-family: Nunito;
+}
+
+.header-closure {
+  color: #4c4c4c;
+  font-size: 14px;
+}
+
+.header-closure a {
+  font-size: 12px;
+}
+.container-content {
+  display: flex;
+  justify-content: center;
+  background: #eceef1;
+}
+
+.inputs {
+  display: inline-block;
+  width: 70%;
+  text-align: left;
+}
+
+.inner {
+  width: 380px;
+  text-align: center;
+  background: #fc8b83;
+}
+
+.inputs input[type="email"],
+.inputs input[type="tel"],
+.inputs input[type="password"] {
+  height: 34px;
+  width: 320px;
+  border: #c1c5c8 solid 1px;
+  border-radius: 4px;
+}
+
+.inputs label {
+  display: inline-block;
+  margin-top: 22px;
+  font-weight: 400;
+  color: #404040;
+  font-size: 14px;
+  line-height: 25px;
+}
+
+.phone-message span {
+  font-size: 10px;
+  color: #444;
+}
+
+header {
+  padding: 20px 0px;
+  background: #eceef1;
+}
+
+header .main-icon {
+  padding: 4px 0px;
+}
+
+button {
+  background: linear-gradient(to bottom, #0077c5 0, #167ccc 100%);
+  border-radius: 4px;
+}
+
+fieldset button {
+  width: 320px;
+  height: 44px;
+  border: none;
+
+  color: white;
```

Comment:
check the linter fails, for example here is failing because of the white space. You could use some CSS [formatter](https://codebeautify.org/css-beautify-minify/21ce48) to be faster with that.